CHANGELOG.md

```diff
@@ -2,6 +2,87 @@
 
 <!-- markdownlint-disable MD024 MD034 -->
 
+## [v1.18.0](https://github.com/FerretDB/FerretDB/releases/tag/v1.18.0) (2024-01-08)
+
+### What's Changed
+
+#### Capped collections
+
+This release adds support for capped collections.
+They can be created as usual using `create` command.
+Both `max` (maximum number of documents) and `size` (maximum collection size in bytes) arguparametersments are supported.
+
+#### Tailable cursors
+
+This release adds support for tailable cursors.
+Both `tailable` and `awaitData` parameters are supported.
+
+#### OpLog tailing
+
+This release adds support for the basic OpLog functionality.
+The main supported use case is Meteor's OpLog tailing.
+Replication is not supported yet.
+
+It is not enabled by default; to enable it, create a capped collection `oplog.rs` in the `local` database.
```

Comment:
Maybe this helps clarity:

```suggestion
OpLog tailing is not enabled by default; to enable it, create a capped collection `oplog.rs` in the `local` database.
```

Reply:
Rewrote the whole sentence (technically, it is not even "OpLog _tailing_")